argocd/overlays/rosa/applications/envs/rosa/nexodus/apex-qa.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
I think it's good if we can change the name of file from apex-qa.yaml to nexodus-qa.yaml